addon/globalPlugins/EnhancedDictionaries/profileConfigurationHelper.py

```diff
@@ -0,0 +1,32 @@
+import config
+from logHandler import log
+
+module = 'EnhancedDictionaries'
+key = "keepUpdatedCheckbox"
+
+
+def stringToBool(value):
+	if value == "True":
+		return True
+	return False
+
+
+def getSavedCheckboxValueForProfile():
+	if module not in config.conf:
+		log.info("o módulo não foi encontrado na configuração")
```

Comment:
Log message suggestion
```suggestion
		log.debug(f"{module} not found on profile's configuration file")
```